index.bs

```diff
@@ -8003,9 +8009,9 @@ Constructors</h4>
 		2. If the {{MediaStreamTrackAudioSourceOptions/mediaStreamTrack}}'s
 			<code>kind</code> attribute is not <code>"audio"</code>, throw an
 			{{InvalidStateError}} and abort these steps.
-		3. Let <var>node</var> be a new {{MediaStreamTrackAudioSourceNode}} object.
-			<a href="#audionode-constructor-init">Initialize</a> <var>node</var>, and
-			return <var>node</var>.
+		3. Let <var>o</var> be an empty dictionnary.
+		4. <a href="#audionode-constructor-init">Initialize the AudioNode</a>
+			<var>this</var>, with <var>c</var> and <var>option</var> as arguments.
```

Comment:
Here, `options` can be passed directly to the "initialize" algorithm, as the mediaStreamTrack member will be ignored.